scene/resources/mesh_library.cpp

```diff
@@ -29,6 +29,7 @@
 /*************************************************************************/
 
 #include "mesh_library.h"
+#include "box_shape_3d.h"
```

Comment:
Nitpick but to match our preferred style from https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes
```suggestion

#include "box_shape_3d.h"
```

Reply:
Done